### PR TITLE
Start Plone 6.1 upgrade guide

### PR DESCRIPTION
**Pull request by stevepiercy**
This is the start of the Plone 6.1 upgrade guide. Please feel free to push commits to this PR and branch.

If you have snippets that I can copy-paste and add to this PR, please let me know.

This PR can wait to be merged until the final release of Plone 6.1, or it can be merged whenever y'all think it is appropriate.

Thank you!

### CODE REVIEW

**Comment by stevepiercy** [comment]
Thanks @1letter! This is super helpful. I polished it up.

File: docs/backend/upgrading/version-specific-migration/upgrade-to-61.md
Comment (davisagli) [inline docs/backend/upgrading/version-specific-migration/upgrade-to-61.md]: This is a bit unclear: If I maintain a Plone site, do I need to take the time to read and understand the info in these links, or were the necessary steps already taken by the Plone classic UI maintainers?

Also this section should mention that the change is only relevant for sites using the classic UI.

Comment (petschki) [reply docs/backend/upgrading/version-specific-migration/upgrade-to-61.md]: Yes this needs more details. Something like:

There's no action needed if you upgrade a standard Classic-UI 6.0 Site to 6.1 (=TinyMCE 7) unless you have custom plugins implemented or you use a plugin which got removed (or moved to premium) in Tiny 6/7. To upgrade your plugin implementation to Tiny 7 please check the upgrade guides here https://www.tiny.cloud/docs/tinymce/6/migration-from-5x/#plugins

Comment (stevepiercy) [reply docs/backend/upgrading/version-specific-migration/upgrade-to-61.md]: Revised in https://github.com/plone/documentation/pull/1655/commits/2b66e7ad90476ae38058db08d4f002798e45a67b. Please take another look.

**Review by davisagli — DISMISSED** [review]

**Comment by stevepiercy** [comment]
Before I merge, are there any other topics that need to be added to the Plone 6.1 Upgrade Guide? I'd be happy to have just the topics, if not the content for them, at this time, and we can do additional PRs to fill in the outline. @mauritsvanrees @sneridagh @pbauer @jensens @petschki.

**Comment by petschki** [comment]
One additional topic would be the changes in `z3c.form` and `plone.app.z3cform` ... addon developers might want to adjust some deprecated imports there. I'll come up with more content in a separate PR and some explanation about the new widget templates here https://github.com/plone/plone.app.z3cform/pull/181

**Review by petschki — DISMISSED** [review]

**Comment by stevepiercy** [comment]
Thanks, @petschki! I added a placeholder in https://github.com/plone/documentation/pull/1655/commits/e71f735bdbc17d8942e8f1c1214996a59c264080, and created a new issue from your comment in https://github.com/plone/documentation/issues/1659.

**Review by jensens — DISMISSED** [review]

**Comment by stevepiercy** [comment]
I'll wait until Sun, May 5, before merging this PR to give everyone a last chance to add topics for the Plone 6.0 to 6.1 Upgrade Guide, or to add their approval if they have none at this time.

**Review by ericof — DISMISSED** [review]

**Comment by mauritsvanrees** [comment]
I have added sections for dropping Python 3.8 and 3.9, and making plone.app.multilingual a core add-on.

**Review by mauritsvanrees — DISMISSED** [review]
So far so good.  Thanks for starting this!

**Comment by stevepiercy** [comment]
Thank you @mauritsvanrees!

**Pull request merged**
